docs/platforms/flutter/upload-debug.mdx

```diff
@@ -157,16 +161,16 @@ You will need to upload the following files:
 
 Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. [Learn how to upload the dSYM files](/platforms/apple/dsym/).
 
-### Android
+### Android, Windows and Linux
 
 See our docs on uploading [Debug Information Files](/cli/dif/#uploading-files) manually with the Sentry CLI.
 
-### Android NDK
-
-See our docs on uploading [Debug Information Files](/cli/dif/#uploading-files) manually with the Sentry CLI.
+<Alert>
 
 If you're using a version of `sentry_flutter` earlier than 5.1, native symbolication on Android requires a specific configuration. Refer to [Troubleshooting](/platforms/flutter/troubleshooting/#native-symbolication-on-android) for more information.
 
+</Alert>
+
```

Comment:
While we're touching this - maybe worth dropping this altogether as it refers to a four year old SDK version and it's unlikely anyone with that version would need to figure this out now...